mlflow/utils/environment.py

```diff
@@ -45,7 +46,7 @@
     r"(?P<version>[\d.]+)?$"
 )
 
-_IS_UNIX = os.name != "nt"
+_IS_UNIX = not is_windows()
```

Comment:
nit:  Can we replace this constant and directly run `is_windows()`. I think the intention of this constant is to avoid duplication, which is no longer be a concern with `is_windows()` function.

Reply:
@B-Step62 TYSM, it's updated now

Reply:
Merged! Thank you so much for your contribution!